database/102-up-timescale/20240507155504-total_volume_adjust_usd_price.sql

```diff
@@ -0,0 +1,67 @@
+-- migrate:up
+
+-- token_prices to contain the current price in US dollars of a token, so that its volume can be correctly scaled
+-- if a token is found in user_actions but not token_prices, its price is assumed to be 1 USD.
+CREATE TABLE token_prices (
```

Comment:
Make this non nullable, and change the name to indicate that this is something temporary until we have a warehouse for this historical data.